ui/src/app/services/workflow/workflow.service.spec.ts

```diff
@@ -58,4 +59,17 @@ describe('WorkflowService', () => {
     expect(req.request.method).toEqual('GET');
     req.flush([...projects]);
   });
+
+  it('getWorkflow() should return projects', () => {
```

Comment:
typo projects should be sth like workflow models?

Reply:
fixed